cfme/tests/candu/test_utilization_metrics.py

```diff
@@ -129,6 +134,7 @@ def test_raw_metric_vm_cpu(metrics_collection, appliance, provider):
             vm_name)
         average_rate = attrgetter('cpu_usage_rate_average')
 
+    pytest.set_trace()
```

Comment:
Is this for debugging purposes? Still necessary?

Reply:
Removed the trace statement.